app/tailwind.config.js

```diff
@@ -5,11 +5,60 @@ module.exports = {
   darkMode: 'class',
   // Default theme: https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L7
   theme: {
-    extend: {
-      colors: {
-        primary: colors.indigo, // primary theme color
-        secondary: colors.yellow, // secondary theme color
-      },
+    fontFamily: {
+      sans: ['"Source Code Pro"', 'monospace'],
+    },
+    fontSize: {
+      sm: ['100%', '170%'],
+      md: ['110%', '170%'],
+      lg: ['120%', '170%'],
+      xl: ['130%', '170%'],
+      xxl: ['140%', '170%'],
+      headline: ['150%', '170%'],
+    },
```

Comment:
Everything in the app feels really big. The font sizes, button sizes, etc. feel like I've zoomed in on the page. Is this correct/intentional/final?

Reply:
maybe your "base font size" is setup weired.  the idea is to us the browsers base-font-size of 12 - 16px ....  and from here just add % of that to various breakpoints.  so for mobile  100% of base font size .  for xxl 140% .    worked nice on all my test devices so far.  

it maybe helps to set the base-fontsize on the html element to 12-16px .

Reply:
Hm, yea we might want to set base fontsize. I don't know if things looked really big for anyone else, but I've never changed my browser font settings 🤷

cc @gdixon

Reply:
Wouldn't adjusting the defined font-sizes make more sense than adjusting the base-font-size? If we set a base font then we lose the ability for our user to define their own?

Reply:
I'm not actually sure which approach is considered best practice. Adjusting the font size percentages here feels like the way to go, but I thought maybe it wasn't per Richard's comment

Reply:
if a user tweaks its own base font size (what less then 0,1% of users do) , they also know how to use cmd+ and cmd- . can not see any issue but wana realy see screenshot what mds1 defines as "to big" as maybe its not even a bug but intentional ...